src/Skoruba.IdentityServer4.STS.Identity/Controllers/AccountController.cs

```diff
@@ -540,6 +540,78 @@ public async Task<IActionResult> LoginWith2fa(LoginWith2faViewModel model)
             return View(model);
         }
 
+        [HttpGet]
```

Comment:
This is the new route on the STS.Identity project that handles the link sent in the e-mail when a user is invited to complete their registration.. It is an Anonymous get, but is careful to ensure it can not be abused by only allowing users that have a valid invitation_token claim, can verify their e-mail address, and have not previously established a password.  If any of these are false they will be directed to the registration process (or the message that registration is disabled based on configuration).  If all is successful, the invitation_token claim is removed once user successfully finishes the registration.